osbenchmark/paths.py

```diff
@@ -22,12 +22,39 @@
 # specific language governing permissions and limitations
 # under the License.
 import os
-
+import sys
+from osbenchmark.utils.io import ensure_dir, ensure_symlink
 
 def benchmark_confdir():
     default_home = os.path.expanduser("~")
-    return os.path.join(os.getenv("BENCHMARK_HOME", default_home), ".benchmark")
+    old_path = os.path.join(default_home, ".benchmark")
+    new_path = os.path.join(default_home, ".osb")
+
+    try:
+        # Ensure .benchmark directory exists
+        ensure_dir(old_path)
+
+        # Ensure symlink from .osb to .benchmark
+        ensure_symlink(old_path, new_path)
+
+        final_path = os.path.join(os.getenv("BENCHMARK_HOME", default_home), ".osb")
```

Comment:
Nit: could be more descriptive with the variable name, would rename `final_path` to `benchmark_confdir_path`.